data-serving/data-service/README.md

```diff
@@ -0,0 +1,8 @@
+# Starting a dev server
+1. Run a local MongoDB instance
+   `mongod`
```

Comment:
Perhaps add a `npm script` for this to `package.json`?

E.g.,

`local-db: mongod && mongoimport --db dev --collection covid19 --file data-serving/samples/line-list.json`

Reply:
added one for importing the data, but mongod stays running in the fg... is there a nice way to script: (1) starting mongod, (2) in a separate process, running mongoimport, (3) killing mongod? or backgrounding mongod & then killing after

Reply:
sg, thanks!